arch/x86/kvm/vmx/pkvm_host.c

```diff
@@ -185,6 +185,26 @@ static int pkvm_vcpu_create(struct kvm_vcpu *vcpu)
 	return ret;
 }
 
+static void pkvm_vcpu_free(struct kvm_vcpu *vcpu)
```

Comment:
another nit about commit message: "They are
freed along with VMX specific resources such as VMCS page, MSR bitmap,
PML page and VE page."

\- again, not the last two? :)